src/librustdoc/clean/auto_trait.rs

```diff
@@ -423,8 +408,8 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
     // K', we use the dedicated syntax 'T: Fn() -> K'
     // * We explicitly add a '?Sized' bound if we didn't find any 'Sized' predicates for a type
     fn param_env_to_generics(
-        &self,
-        cx: &mut DocContext<'tcx>,
+        &mut self,
+        tcx: TyCtxt<'tcx>,
```

Comment:
Why does this take `tcx`? It seems like it uses `self.cx.tcx` in the body.

Reply:
Is it just because it took `tcx` before?

Reply:
> Is it just because it took tcx before?

Yes, I was trying to keep the churn to a minimum.

Reply:
That's fine, but I feel a bit confused—why is CI passing if the argument is unused? Or is it used somewhat in the body?

Reply:
Yes, it's used even though you can get it from cx.tcx.

Reply:
Okay, we can remove that later then.

Reply:
Opened #82137.